.circleci/config.yml

```diff
@@ -236,6 +242,8 @@ workflows:
           matrix:
             parameters:
               version:
+                - v2.0.0
+                - v2.3.9
                 - v2.4.0
-                - v2.5.0-alpha.1
+                - v2.5.0-alpha.4
```

Comment:
is this something we would have to keep changing?

Reply:
no, once GA is done we can leave it. The changes in etc/testing/circle/run_api_performance_tests.sh should let us automatically pick up the latest version more reliably.  So we only need to add new major versions after they're released. We could probably have an improvement to get the latest patch for each version, but I think that is out of scope for this.